src/Shared/Buffers.MemoryPool/MemoryPoolFactory.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Buffers
 {
-    internal static class MemoryPoolFactory
+    internal static class SlabMemoryPoolFactory
```

Comment:
LOL

Reply:
You can't say I didn't warn you :trollface: